server-common/build.gradle.kts

```diff
@@ -39,6 +39,9 @@ dependencies {
   implementation(libs.bundles.metrics)
   implementation(libs.commons.lang3)
   implementation(libs.guava)
+  implementation(libs.jackson.datatype.jdk8)
+  implementation(libs.jackson.datatype.jsr310)
+  implementation(libs.jackson.databind)
```

Comment:
If we add the jackson dependencies here in this module, shall we remove them in Lineage module? Since that module will depend on this module.

Reply:
Yes, removed from the lineage module.